android/src/main/java/com/horcrux/svg/RenderableViewManager.java

```diff
@@ -1284,6 +1284,8 @@ public void onChildViewAdded(View view, View view1) {
             @Override
             public void onChildViewRemoved(View view, View view1) {
                 if (view instanceof VirtualView) {
+                    SvgView svgView = ((VirtualView) view).getSvgView();
+                    svgView.setRemovedFromReactViewHierarchy();
```

Comment:
I tested your code and there is a chance that `svgView` is null. I would suggest doing a null check here - just to avoid potential crashes.